docs/_tutorials/local_development.md

```diff
@@ -71,40 +59,15 @@ Once you've installed a development proxy, run it to begin forwarding requests t
 
 The output should show a generated URL that you can use (we recommend the one that starts with `https://`). This URL will be the base of your request URL, in this case `https://8e8ec2d7.ngrok.io`.
 
-### Add the URL to your app configuration
-Okay, so hopefully at this point you have some kind of public-facing URL. Now, let's add that to the **Interactive Components** page. Under the **Request URL** box, go ahead and paste in your accessible URL.
-
-If you're using the default HTTP server in the `@slack/interactive-messages` package, you should append `/slack/actions` to your URL. For our example, this will be `https://8e8ec2d7.ngrok.io/slack/actions`. If you are using the Express middleware, you can set whichever path you like, just remember to make the path you mount the middleware into the application the same as the one you configure in Slack.
+#### Add the URL to your app configuration
+At this point you have a public-facing URL. Now, let's add that to the **Interactive Components** page. Under the **Request URL** box, go ahead and paste in your accessible URL.
 
 ![Configuring a Request URL](request-url-config.png "Configuring a Request URL")
 
-## Listening to workspace events
+### Listening to workspace events
```

Comment:
```suggestion
### Subscribing to workspace events
```
The instructions here seem more relevant to Subscribing your app to an event vs. setting up your event handler/listener code (for example in Bolt "listener" [means the handling code](https://slack.dev/bolt-js/concepts#event-listening)")